mappings/net/minecraft/item/ArmorMaterial.mapping

```diff
@@ -1,11 +1,77 @@
 CLASS net/minecraft/class_1741 net/minecraft/item/ArmorMaterial
+	COMMENT Defines the material stats of an {@link net.minecraft.item.ArmorItem} item.
```

Comment:
There's no need to fully qualify `ArmorItem`, `ArmorMaterials` and `Item(.Settings)` since they're in the same package.

Reply:
That's good to know, thank you!